app/common/directives/dropdown.js

```diff
@@ -281,6 +281,29 @@ angular.module('ushahidi.common.dropdown', ['ui.bootstrap.position'])
     };
 })
 
+.directive('isFloating', function () {
```

Comment:
What does isFloating do? If this is an parameter to `dropdown` it should probably get handled in the drop down directive via `$attr`